### PR TITLE
VT queued files

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22197

## Description
When sending a file to VirusTotal and instantly querying it, VT would return a status -2 that means that the file is in VT but is waiting for analysis. This status wasn't handled properly.

## Does it break backward compatibility?
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

- [ ] [CHANGELOG](https://github.com/demisto/content/edit/Virustotal_none_string/Integrations/VirusTotal-Private_API/CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5834)

### CODE REVIEW

**Comment by kirbles19** [comment]
@orlichter1 Done.

File: Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py [hunk 1]
```diff
@@ -1115,35 +1136,44 @@ def download_file_command():
...
+            demisto.results(get_ip_report_command())
+        elif demisto.command() == 'vt-private-search-file':
+            demisto.results(search_file_command())
+        elif demisto.command() == 'vt-private-hash-communication':
+            demisto.results(hash_communication_command())
+        elif demisto.command() == 'vt-private-download-file':
+            demisto.results(download_file_command())
+
+    except Exception as e:
+        LOG(e.message)
```
Comment (IkaDemisto) [inline Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py]: use `return_error` instead

File: Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py [hunk 2]
```diff
@@ -1115,35 +1136,44 @@ def download_file_command():
...
+
+def main():
+    LOG('command is %s' % (demisto.command(),))
+    try:
+        handle_proxy(proxy_param_name='useProxy')
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration test button.
+            # disable-secrets-detection-start
+            if check_file_behaviour(
+                    '10676cf66244cfa91567fbc1a937f4cb19438338b35b69d4bcc2cf0d3a44af5e'):  # guardrails-disable-line
```
Comment (IkaDemisto) [inline Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py]: why we need this?

File: Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py [hunk 3]
```diff
@@ -1115,35 +1136,44 @@ def download_file_command():
...
+                # disable-secrets-detection-end
+                demisto.results('ok')
+            else:
+                demisto.results('test failed')
+            pass
```
Comment (IkaDemisto) [inline Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py]: looks not necessary

File: Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py (hunk 2 above)
Comment (orlichter1) [reply Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py]: I'm not sure. Was here before me. I think just a general file to check and get an answer.

**Pull request merged**